docs/content/ingestion/schema-design.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: doc_page
+---
+
+# Schema Design
+
+This page is meant to assist users in designing a schema for data to be ingested in Druid. Druid intakes denormalized data 
+and columns are one of three types: a timestamp column, a dimension, or a measure (or a metric/aggregator as they are 
+known in Druid). This follows the [standard naming convention](https://en.wikipedia.org/wiki/Online_analytical_processing#Overview_of_OLAP_systems) 
+of OLAP data.
+
+Typical production tables (or datasources as they are known in Druid) have less than 100 dimensions and less 
+than 100 metrics, although, based on user testimony, datasources with thousands of dimensions have been created. At the 
+time of writing, we have not benchmarked the upper limits of how many columns Druid is capable of.
+
+Dimension values in Druid can either be a single valued String, or an array of String values. Metric columns can be 
+integers, longs, floats, or in a user defined 'complex' format. Metric values may be [rolled-up](../design/index.html) 
+data at ingestion time. 
+
+Below, we outline some best practices with schema design:
+
+## High cardinality dimensions (e.g. unique IDs)
+
+In practice, we see that exact counts for unique IDs are often not required. Storing unique IDs as a column will kill 
+roll-up ratios, and impact compression. Instead, storing a sketch of the number of the unique IDs seen, and using that 
+sketch as part of aggregations, will greatly improve performance (up to orders of magnitude performance improvement), and significantly reduce storage. 
+Druid's `hyperUnique` aggregator is based off of Hyperloglog and can be used for unique counts on a high cardinality dimension.
+
+## Nested dimensions
+
+At the time of this writing, Druid does not support nested dimensions. Nested dimensions need to be flattened. 
+
+## Counting the number of ingested events
+
+A count aggregator at ingestion time can be used to count the number of events ingested. However, it is important to note 
+that when you query for this metric, you should use a `longSum` aggregator. A `count` aggregator at query time will return 
+the number of Druid rows for the time interval, which can be used to determine what the roll-up ratio was.
+
+## Schema-less dimensions
+
+If the `dimensions` field is left empty in your ingestion spec, Druid will treat every column that is not the timestamp column, 
+a dimension that has been excluded, or a metric column as a dimension. It should be noted that because of [#658](https://github.com/druid-io/druid/issues/658) 
+these segments will be slightly larger than if the list of dimensions was explicitly specified in lexicographic order. This limitation 
+does not impact query results as all counts should still be correct.
```

Comment:
replace "does not impact query results as all counts should still be correct" with "does not impact query correctness- just storage requirements"